TexasHoldem.c

```diff
@@ -1,7 +1,456 @@
 //Write the texas holdem game here
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
+#include <math.h>
 
+
+//function prototypes
+//i put a little bit more detail at the bottom for each function's definitions
+//displays opening directions
+void Directions(void);
+//runs game(same idea as in Connect4 to clear up main)
+void RunTexasHoldem(void);
+//chooses how many players are playing
+int playerChoice(void);
+//betting function(still working on)
+void DefineBet(int, double[]);
+//randomCard draws a card and CheckRand makes sure the card drawn isn't already drawn
+void randomCard(int, int, int[][2], int[][2], int[][2], int[][2]);
+int CheckRand(int, int, int[][2], int[][2], int[][2], int[][2]);
+
+//randomCommunityCard draws community cards and CheckRandCommunity makes sure the community cards drawn aren't already drawn
+void randomCommunityCard(int, int[][2]);
+int CheckRandCommunity(int, int[][2]);
+
+//main
 int main() {
+	//seeding for random number
+	unsigned seed = 7;
+	srand(seed);
+	
+	Directions();
+	RunTexasHoldem();
+	
+	return 0;
+}
```

Comment:
Everything here looks good, only change i would consider making is adding a time library and using that for the seed